packages/nova-lib/lib/redux.js

```diff
@@ -1,2 +1,3 @@
 export const Actions = {};
 export const Reducers = {};
+export const Middleware = {};
```

Comment:
"Middleware" is correct for both singular and plural in English, though [the term is not without it's detractors](https://github.com/rack/rack/issues/332).